landIceMeshGen.cc

```diff
@@ -254,8 +339,9 @@ int main(int argc, char **argv)
   pGFace* faces;      // array to store the returned model faces
   pGRegion region;      // pointer to returned model region
   pGModel model;        // pointer to the complete model
-
-  auto dirty = readJigGeom(argv[1]);
+  
+  auto dirty = readVtkGeom(argv[1]);
+  //auto dirty = readJigGeom(argv[1]);
```

Comment:
Would you please add a conditional on the extension of the input file so the correct reader is called?  For now, I think keeping the jigsaw loader is a good idea.